nyx_client/cli/init_env.py

```diff
@@ -35,6 +35,15 @@ def init_env(filename: str = ".env"):
             click.echo("Exiting with no changes")
             return
 
+    # Check user is aware of T&Cs
+    tcs = input(
+        "The Nyx SDK allows the sharing of links through a Nyx Instance. "
```

Comment:
Is this enough? or do we want to call out each point as in UI?

Reply:
Valid point, I think copying the UI makes sense, even if it's just for consistence

Reply:
https://github.com/Iotic-Labs/nyx-sdk/pull/20/commits/81377cffa14091585ae573bccbbe71f62609d504